ddd-koans/Value Object Pt. 2/Terms Dictionary/task.html

```diff
@@ -0,0 +1,49 @@
+<html>
+<h2>Domain terms</h2>
+<div>
+    <ul>
+        <li> <b>Time signature</b> - Symbols that describe repeating beat rhythm in a piece of music.
+            It consists of the number of beats and the note value.
+        </li>
+        <li> <b>Number of beats</b> -
+            The upper number of time signature indicates how many beats constitute a bar.
+        </li>
+        <li> <b>Note value</b> -
+            The lower number of time signature always corresponds to the beat unit which is a value of a music note.
+        </li>
+        <li> <b>Bar</b> -
+            A written up piece of music is divided into chunks called bars.
+            Each bar follows a time signature assigned at the beginning of the bar until a new time signature occurs.
+        </li>
+        <li> <b>Note</b> -
+            Symbol is denoting a musical sound. Notes can represent the pitch and duration of a sound in musical notation.
+        </li>
+    </ul>
+    <!--    TODO("Describe rest of the domain terms")     -->
```

Comment:
should I explain sth here?

Reply:
This is a place where we sum up all of the terms from this part and I just would like to do it in the future.

Reply:
This is just now a copy-paste from the previous lesson and needs to be updated with terms from this VO part 2 lesson.